classes/fields/number.php

```diff
@@ -82,6 +82,16 @@ public function options() {
 				'type'        => 'boolean',
 				'excludes-on' => array( static::$type . '_decimals' => 0 ),
 			),
+			static::$type . '_keep_leading_zeroes' => array(
```

Comment:
I would choose a `_format` field instead of a specific field for this case just as we dit with the currency field.

https://github.com/pods-framework/pods/blob/3.0.2/classes/fields/currency.php#L100-L133

Note the decimals and decimal_handling fields below the format options.

Reply:
Can you clarify what you mean here? do you mean use `_format` as part of the option name?

```suggestion
			static::$type . '_format_keep_leading_zeroes' => array(
```

Reply:
I think we could add this to currency field too, but I can't think of a legitimate use case for a currency with leading zeroes. Unless that's really a thing?

Reply:
What I meant is that the currency field already supports this and I think it's better to keep the code the same as these fields inherit from each other. In this case, the currency field inherits from the number field.

Reply:
Hold on, I totally misunderstood the option hear, I was thinking about trailing zero's, not leading...